vizro-core/src/vizro/models/_components/table.py

```diff
@@ -27,8 +27,9 @@ class Table(VizroBaseModel):
     table: CapturedCallable = Field(..., description="Table to be visualized on dashboard")
     actions: List[Action] = []
 
+    _datatable_id: str = PrivateAttr()
+
     # Component properties for actions and interactions
-    _input_property: str = PrivateAttr("active_cell")
```

Comment:
Removed since there are two input properties ("active_cell" and "derived_viewport_data") for `dash_data_table` objects. And for the `AgGrid` implementation there are probably some other properties.